build/config/PublishData.json

```diff
@@ -0,0 +1,36 @@
+{
+  "master": {
+    "nugetPush": [
+        {
+            "kind": "PerBuildPreRelease", 
+            "url": "https://dotnet.myget.org/F/roslyn/api/v2/package"
+        }
+    ],
+    "vsixPush": "https://dotnet.myget.org/F/roslyn/vsix/upload",
+    "channels": [
+        {
+            "name": "dev15.5",
+            "kind": "PerBuildPreRelease"
```

Comment:
Example for 15.3 right now. 

nugetPush: PerBuildPreRelease and Release
dev15.3 RTM packages
dev15.3p5 beta packages